### PR TITLE
Rename binlog proto

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve? 

To solve the protocol buffer namespace conflict problem.

### What is changed and how it works?

Rename binlog.proto to bl.proto

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

**Review by glorv — APPROVED** [review]

File: tidb-binlog/proto/go-binlog/bl.pb.go [hunk 1]
```diff
@@ -1,11 +1,11 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
-// source: binlog.proto
+// source: bl.proto
```
Comment (lichunzhu) [inline tidb-binlog/proto/go-binlog/bl.pb.go]: Name this to `slave_binlog.proto` or `secondary_binlog.proto`? `bl.proto` is quite strange.

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

**Review by lichunzhu — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]

**Pull request merged**